backend/src/main/java/com/zeros/backend/api/RestaurantSubscriptionResource.java

```diff
@@ -0,0 +1,64 @@
+package com.zeros.backend.api;
+
+import com.zeros.backend.exceptions.AlreadySubscribedRestaurantException;
+import com.zeros.backend.models.Restaurant;
+import com.zeros.backend.services.RestaurantSubscriber;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/api/v1")
+public class RestaurantSubscriptionResource {
+
+    @Autowired
+    private RestaurantSubscriber restaurantAffiliateNetworwk;
```

Comment:
You still have a typo here @JumperSpirit. You can name it restaurantSubscriber instead I thinks that it sounds better.